services/pkgreview/package-rating.tester.js

```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+const t = (module.exports = require('../tester').createServiceTester())
+
```

Comment:
Similarly to the helpers for the service class, we've got a lot of helpers for test validation as well. If you import these suggseted ones, then the they can be used to complete these tests

```suggestion
const { withRegex, isStarRating } = require('../test-validators')

const isRatingWithReviews = withRegex(/[0-5]\.[0-9]{1}\/5?\s*\([0-9]*\)$/)

```

Reply:
I tried this in commit `88d64b517d5fb5a15280872a04196a0391544249`... But for a reason, it fails.

According to CircleCI logs, `5/5 (3)` does not match the regex.

Reply:
Well, the regex `\^[0-5](.?([0-9]))?\/5?\s*\([0-9]*\)$\` matches `5/5 (3)`.

There's just one bug with this... that is... even `5.5/5 (3)` matches though the maximum is 5.

Reply:
Yup, sorry about that. Copied from a different service and needs a different regex, see my response in https://github.com/badges/shields/pull/4660#issuecomment-589889940 which has one that should work here